capstone/pom.xml

```diff
@@ -81,6 +81,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>1.108.0</version>
+    </dependency>
```

Comment:
You should not need this dependency -- `spring-cloud-gcp-starter-storage` brings a compatible version transitively.

Reply:
The policy is in this dependency so I still need to include it.

Reply:
@elefeint `PostPolicyV4` doesn't exist in the older version of `google-cloud-storage` that is pulled in transitively.